GoogleForCreators/web-stories-wp
# Define block icon as React component

## Context

The block icon could have had broken gradients, see #8401

## Summary



Moves the block icon to a React component to ensure unique IDs for each instance of the icon.

## Relevant Technical Choices



## To-do



## User-facing changes



![Screenshot 2021-07-23 at 14 25 36](https://user-images.githubusercontent.com/841956/126781373-c09a3166-e4f4-447f-ad46-0b2b0bf71bfe.png)


## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Go to Appearance -> Customizer to insert the Web Stories block a few times.
2. Ensure the block icon in the block toolbar is always visible and presented correctly.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #8401
